pkg/sampler/sampler.go

```diff
@@ -57,65 +57,56 @@ func (e StatusCodeError) Error() string {
 	)
 }
 
-// Sampler implements Sampler, using http.Transport.
-type Sampler struct {
-	tr        http.Transport
-	UserAgent string
-}
+// Ping measures a given URL and returns a Sample
+func Ping(target Target, timeout int) (sample Sample, err error) {
```

Comment:
Ha!  I was struggling with that one.  I can't use `Sample` (my first choice) because this package already has a type named `Sample`.

Happy to try something else - what's your next favorite suggestion?  Naming things is hard.